src/components/Layout.tsx

```diff
@@ -41,7 +48,14 @@ export function Layout() {
           alignContent={'flex-start'}
           flexBasis={'50vh'}>
           {sidebar?.visible ? <Sidebar /> : null}
-          <Grid item bgcolor={grey['100']} padding={3} height={'100%'} xs>
+          <Grid
+            item
+            bgcolor={grey['100']}
+            padding={3}
+            height={'100%'}
+            xs
+            paddingLeft={sidePadding}
+            paddingRight={sidePadding}>
```

Comment:
We can use just paddingX prop instead of paddingLeft and paddingRight

Reply:
good point